### PR TITLE
refactor: report design modification

### PR DESCRIPTION
**Pull request by stevenyh3**

### CODE REVIEW

**Review by cabreraalex — COMMENTED** [review]
small thing: I actually find the opacity changes to the bar chart to be a bit distracting, could we just show the tooltip and not change the opacity? I think it's fine for the line chart.

File: frontend/src/report/report-page/report-header/ReportHeader.svelte [hunk 1]
```diff
@@ -42,8 +42,8 @@
 			variant="outlined"
 			bind:value={currentReport.name}
 			label="Report Name"
-			input$maxlength={15}>
-			<CharacterCounter slot="helper">0 / 15</CharacterCounter>
+			input$maxlength={30}>
+			<CharacterCounter slot="helper">0 / 30</CharacterCounter>
```
Comment (cabreraalex) [inline frontend/src/report/report-page/report-header/ReportHeader.svelte:46]: Do we need a max length at all? I guess this is good for rendering to make sure it looks nice in the titles and stuff. I'm okay with it I think

**Comment by cabreraalex** [comment]
Only main issue: Now when I change the chart type it automatically changes the report name. Definitely don't think it should do this, it should just keep the original name, we can just tick with things like "New Report" I think

**Comment by cabreraalex** [comment]
Comment on colors: I like the purples but I think having a diverging color scale can be confusing and misconstrued for showing a change in magnitude. I think for the colors we should maybe stick to something more traditional like category 20: https://vega.github.io/vega/docs/schemes/#category20

**Comment by cabreraalex** [comment]
Can we rename slice-chart to bar-chart?

File: frontend/src/report/report-page/report-header/ReportHeader.svelte (hunk 1 above)
Comment (stevenyh3) [reply frontend/src/report/report-page/report-header/ReportHeader.svelte:46]: hmm actually I've hidden the overflow, so we could remove the max length, the format should stay the same no matter how many characters the users have input
image

**Pull request merged**